packages/ListBox/src/helpers/applyOnChange.js

```diff
@@ -0,0 +1,49 @@
+import useListBox from "../useListBox";
+
+export function cleanActionTypes() {
```

Comment:
What makes them `clean`?

Reply:
maybe is cleaningActionTypes the name, I'm removing some actions types that are internal and the consumer shouldn't use or might not even interested in using.

Reply:
Ah. Okay, so probably something like `getCleanActionTypes` would be more appropriate as it returns some actionTypes, but isn't a method to mutate the actionTypes.

Reply:
I called sanitizeActionTypes is merged